services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2779,8 +2795,11 @@ public SchemaEntry addValueSchema(
               expectedCompatibilityType);
           // Check if the superset schema already exists or not. If exists use the same ID, else bump the value ID by
           // one.
-          int supersetSchemaId =
-              getVeniceHelixAdmin().getValueSchemaIdIgnoreFieldOrder(clusterName, storeName, newSuperSetSchemaStr);
+          int supersetSchemaId = getVeniceHelixAdmin().getValueSchemaIdIgnoreFieldOrder(
```

Comment:
In this case, will the custom prop superset schema generator carry the custom prop in the newly inserted value schema?

Reply:
It will and as you can see this function is passing a schema comparator, which will consider the custom prop.

And the custom superset generator will consider the custom prop in all these function calls:
```
generateSupersetSchemaFromSchemas
compareSchema
```